### DISCUSSION TITLE
DHCP Relay

### DISCUSSION THREAD

**Discussion by polachz** [TechnitiumSoftware/DnsServer#1129] [Q&A]
Hallo.

I'm trying to use Technitium DNS as my main server in homelab and LAN. But to get best results, I can use it as DHCP server too. I'm using multiple segments here and I can't put many interfaces to the DNS server. I prefer to place the server in special SRV segment and forward queries by my firewall and router to the Technitium DNS server. 

For DNS it works well. But for DHCP I'm not able to do that without DHCP relay. DHCP relay works well for one segment. But is not able to handle more segments without Relay Agent Information (To tag query by info what scope have to be used). But I can't find info if this is supported by Technitium DNS, or if is here any diferent way how to reach my goals.

Thank you for the reply.

**Comment by ShreyasZare** [comment]
Thanks for the post. The DHCP server does not support Relay Agent Information option. The server selects the scope based on the relay agent IP address so if the Relay Agent can be configured to use the IP address of the subnet it received the DHCP request would make it work.

I can get this option supported when the DHCP server is planned for major upgrade but it will take lot of time since there are several DNS features planned to be implemented.
